site/gatsby-site/src/pages/incidents/edit.js

```diff
@@ -6,24 +6,42 @@ import useToastContext, { SEVERITY } from '../../hooks/useToast';
 import { Button } from 'react-bootstrap';
 import { Spinner } from 'flowbite-react';
 import { FIND_INCIDENT, UPDATE_INCIDENT } from '../../graphql/incidents';
+import { UPSERT_ENTITY } from '../../graphql/entities';
 import { useMutation, useQuery } from '@apollo/client/react/hooks';
 import { Formik } from 'formik';
 import { LocalizedLink } from 'gatsby-theme-i18n';
 import { useTranslation, Trans } from 'react-i18next';
+import { graphql, useStaticQuery } from 'gatsby';
+import { processEntities } from '../../utils/entities';
 
 function EditCitePage(props) {
   const { t, i18n } = useTranslation();
 
-  const [incident, setIncident] = useState();
+  const [incident, setIncident] = useState(null);
 
   const [incidentId] = useQueryParam('incident_id', withDefault(NumberParam, 1));
 
   const { data: incidentData, loading } = useQuery(FIND_INCIDENT, {
     variables: { query: { incident_id: incidentId } },
   });
 
+  const {
+    entities: { nodes: allEntities },
+  } = useStaticQuery(graphql`
```

Comment:
This will lag behind what is in the actual DB, is it better to query the Atlas Graphql instead? 🤔

Reply:
Issue created https://github.com/responsible-ai-collaborative/aiid/issues/1354